app/code/core/Mage/CatalogInventory/Model/Stock.php

```diff
@@ -50,6 +50,9 @@ class Mage_CatalogInventory_Model_Stock extends Mage_Core_Model_Abstract
     const STOCK_IN_STOCK            = 1;
 
     const DEFAULT_STOCK_ID          = 1;
+    
+    const EVENT_CORRECT_STOCK_ITEMS_QTY_BEFORE = 'cataloginventory_stock_item_correct_qty_before';
```

Comment:
Being a bit pedantic here but I think a better term than "correct" would be "adjust" since correct implies that it was previously incorrect which could be confusing.